kobuki_node/package.xml

```diff
@@ -1,8 +1,8 @@
-<package>
+<package format="2">
   <name>kobuki_node</name>
   <version>0.7.6</version>
   <description>
-      ROS nodelet for Kobuki: ROS wrapper for the Kobuki driver.
+      ROS node for Kobuki: ROS wrapper for the Kobuki driver.
   </description>
   <author>Daniel Stonier</author>
   <author>Younghun Ju</author>
```

Comment:
Looks like we are missing a maintainer field here too.

```
  <maintainer email="d.stonier@gmail.com">Daniel Stonier</maintainer>
```

Reply:
It's there in line 12, as far as I can tell.